boards/nano33ble/README.md

```diff
@@ -0,0 +1,103 @@
+Arduino Nano 33 BLE
+===================
+
+<img src="https://store-cdn.arduino.cc/usa/catalog/product/cache/1/image/1040x660/604a3538c15e081937dbfbd20aa60aad/a/b/abx00031_featured.jpg" width="35%">
+
+The [Arduino Nano 33 BLE](https://store.arduino.cc/usa/nano-33-ble) and [Arduino
+Nano 33 BLE Sense](https://store.arduino.cc/usa/nano-33-ble-sense) are compact
+boards based on the Nordic nRF52840 SoC. The "Sense" version includes the
+following sensors:
+
+- 9 axis inertial sensor
+- humidity and temperature sensor
+- barometric sensor
+- microphone
+- gesture, proximity, light color and light intensity sensor
+
+
+## Getting Started
+
+First, follow the [Tock Getting Started guide](../../../doc/Getting_Started.md)
+
+You will need the bossac bootloader tool:
+
+```shell
+$ git clone https://github.com/arduino/BOSSA
+$ cd BOSSA
+$ make bossac
+```
```

Comment:
Need to say something along the lines of "You will need to add the `BOSSA/bin/` directory to your PATH."